scripts/modules/task-manager/analyze-task-complexity.js

```
@@ -109,16 +119,42 @@ async function analyzeTaskComplexity(options, context = {}) {
+		}
+
+		// Store header info for later display after we know the task count
+		var headerInfo = {
+			tasksFilePath: tasksPath,
+			outputPath: outputPath,
+			model: model,
+			temperature: temperature,
+			research: useResearch,
+			threshold: thresholdScore,
+			analysisScope: analysisScope
+		};
```

Biome (1.9.4)

[error] This var should be declared at the root of the enclosing function.

The var is accessible in the whole body of the enclosing function.
To avoid confusion, it should be declared at the root of the enclosing function.

(lint/correctness/noInnerDeclarations)